app/actions/remote/user.ts

```diff
@@ -890,12 +890,16 @@ export const fetchCustomAttributes = async (serverUrl: string, userId: string):
 
         if (fields?.length > 0) {
             const attributes: Record<string, CustomAttribute> = {};
-            fields.forEach((field) => {
-                attributes[field.id] = {
-                    id: field.id,
-                    name: field.name,
-                    value: attrValues[field.id] || '',
-                };
+            fields.forEach((field: CustomProfileField) => {
+                const value = attrValues[field.id] || '';
+                if (!filterEmpty || value) {
```

Comment:
Just a sanity check. `value` is always going to be a string, and an empty string is always considered empty. Right? Or is there any distinction between undefined and the empty string to be done here?

Reply:
no, there is no distinction, I just make sure value is always a string